recipe/meta.yaml

```
@@ -1,42 +1,76 @@
 {% set name = "access" %}
-{% set version = "1.1.3" %}
+{% set version = "1.1.9" %}
+{% set gh_src = "github_source_" ~ name ~ "-" ~ version %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 381aecadd7ba70d712b74cd92fb0906e16b362522ae0c7de9b8e2424d19af51d
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 89d0043e3596854701b72c77dfedb677839a5b416a18362c222b7d321ad02138
+
+   # tests and tests data files
+  - url: https://github.com/pysal/access/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: f6cddb394abc4e1fa518d156fd1e6041e3c9b75c788ae7be074962a1ab08116e
+    folder: {{ gh_src }}
 
 build:
-  noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
-    - python >=3.5
+    - python
     - pip
+    - setuptools >=61.0
+    - setuptools-scm >=6.2
+    - wheel
   run:
-    - python >=3.5
-    - scipy >=0.11
+    - geopandas
     - numpy >=1.3
     - pandas >=0.23.4
+    - python
     - requests >=2
 
+# =======================
+# Test configuration
+# =======================
+{% set test_root = gh_src ~ "/access/tests" %}
+
+# - TestEuclideanNeighbors: AssertionError
+{% set skip_keywords = [
+    "TestEuclideanNeighbors"
+] %}
+
 test:
-  # Other tests run through CI.
+  requires:
+    - pip
+    - python
+    - coverage
+    - coveralls
+    - matplotlib
+    - pytest
+    - pytest-cov
+    - pytest-xdist
+    - scipy>=0.11
+  source_files:
+    - {{ test_root }}
   imports:
     - access
+    - access.datasets
+  commands:
+    - pip check
+    - python -c "import access; print(access.__version__)"
+    - pytest -v {{ test_root }} -k "not ({{ skip_keywords | join(' or ') }})"
```

```suggestion
    - pytest -n auto -v {{ test_root }} -k "not ({{ skip_keywords | join(' or ') }})"
```
Since you have pytest-xdist

---

thanks didn't know that